<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Dubai</family>
          <pointsize>21</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(226, 226, 226);

color: rgb(164, 166, 168);</string>
        </property>
        <property name="text">
         <string>Spring Studio 0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QsciScintilla" name="frame_editor">
        <property name="font">
         <font>
          <family>Bradley Hand ITC</family>
          <pointsize>24</pointsize>
          <underline>true</underline>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QMenu{
    padding:5px;
    background:white;
    border:1px solid gray;
 }
QMenu::item{
    padding:0px 60px 0px 60px;
    height:40px;
}
QMenu::item:selected:enabled{
    background:lightgray;
    color:black;
}
QMenu::item:selected:!enabled{
    background:transparent;
}
QMenu::separator{
    height:1px;
    background:lightgray;
    margin:5px 0px 5px 0px;
}</string>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuToolbar_Style">
     <property name="title">
      <string>Toolbar Style</string>
     </property>
     <addaction name="action_toolbar_text"/>
     <addaction name="action_toolbar_icon"/>
     <addaction name="action_toolbar_icon_bottom_text"/>
     <addaction name="action_toolbar_icon_right_text"/>
    </widget>
    <addaction name="action_open_cmd"/>
    <addaction name="menuToolbar_Style"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_file"/>
    <addaction name="action_save"/>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_step_continue"/>
    <addaction name="action_stop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_workspace">
   <property name="windowTitle">
    <string>数据</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_workspace">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_output">
   <property name="windowTitle">
    <string>程序输出</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit_output">
          <property name="font">
           <font>
            <family>宋体</family>
           </font>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_files">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QDockWidget{
border:1px solid lightgray;
}

QDockWidget::title {
text-align: left;/* align the text to the left */
padding-left:5px;
padding-top:5px;
padding-bottom:5px;
padding-right:5px;
	background-color: rgb(211, 211, 211);
border:0px solid lightgray;

	color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>当前文件夹</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget_files">
       <property name="styleSheet">
        <string notr="true">QListView::item {
height:42px;
}</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::EditKeyPressed</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>文件工具栏</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{
background-color: rgb(255, 255, 255);
	background-color: rgb(250, 250, 250);
border:0px solid lightgray;
}

QToolBar:handle {
    border-left:  dotted yellow;
    color: yellow;
}

QToolBar::separator {
    width: 1px;
margin-left: 10px;
margin-right: 10px;
margin-top:1px;
margin-bottom:1px;
background-color: rgb(38, 40, 41);
background-color: lightgray;
}


QToolButton{
    padding:5px 15px 5px 15px;
    border: 0px solid #3daee9;
}

QToolButton:hover, QToolButton::menu-button:hover {
	background-color: lightgray;
    border: 0px solid #3daee9;
    padding:5px 15px 5px 15px;
}
QToolButton:checked, QToolButton:pressed,
        QToolButton::menu-button:pressed {
	background-color: darkgray;
    border: 0px solid #3daee9;
    padding:5px 15px 5px 15px;
}</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new_file"/>
   <addaction name="action_save"/>
   <addaction name="actionOpen"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>调试工具栏</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{
background-color: rgb(255, 255, 255);
	background-color: rgb(250, 250, 250);
border:0px solid lightgray;
}

QToolBar:handle {
    border-left: 0px dotted yellow;
    color: #BBBBBB;
}

QToolBar::separator {
    width: 1px;
margin-left: 10px;
margin-right: 10px;
margin-top:1px;
margin-bottom:1px;
background-color: rgb(38, 40, 41);
background-color: lightgray;
}


QToolButton{
    padding:5px 15px 5px 15px;
    border: 0px solid #3daee9;
}

QToolButton:hover, QToolButton::menu-button:hover {
	background-color: lightgray;
    border: 0px solid #3daee9;
    padding:5px 15px 5px 15px;
}
QToolButton:checked, QToolButton:pressed,
        QToolButton::menu-button:pressed {
	background-color: darkgray;
    border: 0px solid #3daee9;
    padding:5px 15px 5px 15px;
}</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_run"/>
   <addaction name="action_step_continue"/>
   <addaction name="action_stop"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>输出工具栏</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolBar{
background-color: rgb(255, 255, 255);
	background-color: rgb(250, 250, 250);
border:0px solid lightgray;
}

QToolBar:handle {
    border-left: 0px dotted yellow;
    color: #BBBBBB;
}

QToolBar::separator {
    width: 1px;
margin-left: 10px;
margin-right: 10px;
margin-top:1px;
margin-bottom:1px;
background-color: rgb(38, 40, 41);
background-color: lightgray;
}


QToolButton{
    padding:5px 15px 5px 15px;
    border: 0px solid #3daee9;
}

QToolButton:hover, QToolButton::menu-button:hover {
	background-color: lightgray;
    border: 0px solid #3daee9;
    padding:5px 15px 5px 15px;
}
QToolButton:checked, QToolButton:pressed,
        QToolButton::menu-button:pressed {
	background-color: darkgray;
    border: 0px solid #3daee9;
    padding:5px 15px 5px 15px;
}</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_clear_output"/>
   <addaction name="action_clear_data"/>
  </widget>
  <action name="action_run">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_run3.png</normaloff>:/icon/img/ic_run3.png</iconset>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
   <property name="toolTip">
    <string>运行/继续</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_open6.png</normaloff>:/icon/img/ic_open6.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_step_continue">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_step_continue.png</normaloff>:/icon/img/ic_step_continue.png</iconset>
   </property>
   <property name="text">
    <string>步进</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_stop5.png</normaloff>:/icon/img/ic_stop5.png</iconset>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_save10.png</normaloff>:/icon/img/ic_save10.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_clear_output">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_clear_output13.png</normaloff>:/icon/img/ic_clear_output13.png</iconset>
   </property>
   <property name="text">
    <string>清空输出</string>
   </property>
   <property name="toolTip">
    <string>清空输出窗口</string>
   </property>
  </action>
  <action name="action_clear_data">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_clear_data1.png</normaloff>:/icon/img/ic_clear_data1.png</iconset>
   </property>
   <property name="text">
    <string>清空数据</string>
   </property>
   <property name="toolTip">
    <string>清空数据</string>
   </property>
  </action>
  <action name="action_new_file">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_new7.png</normaloff>:/icon/img/ic_new7.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/icon/img/ic_pause4.png</normaloff>:/icon/img/ic_pause4.png</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
   <property name="toolTip">
    <string>暂停</string>
   </property>
  </action>
  <action name="action_toolbar_icon">
   <property name="text">
    <string>图标</string>
   </property>
  </action>
  <action name="action_toolbar_icon_bottom_text">
   <property name="text">
    <string>图标+底部文字</string>
   </property>
  </action>
  <action name="action_toolbar_icon_right_text">
   <property name="text">
    <string>图标+右侧文字</string>
   </property>
  </action>
  <action name="action_toolbar_text">
   <property name="text">
    <string>文字</string>
   </property>
  </action>
  <action name="action_open_cmd">
   <property name="text">
    <string>打开命令行</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header location="global">Qsci/qsciscintilla.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
